services/provider/server/server.go

```diff
@@ -400,6 +416,59 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 
 	}
 
+	// Fetch noobaa remote secret and management address and append to extResources
+	noobaaOperatorSecret := &v1.Secret{}
+	clusterID := strings.TrimPrefix(consumerResource.Name, "storageconsumer-")
+	if clusterID != "" && len(clusterID) == 0 {
+		return nil, fmt.Errorf("failed to get clusterID from consumerResource Name: %s %v", consumerResource.Name, err)
+	}
```

Comment:
this block is already repeated twice. client clusterid now can be directly queried from status section of storageconsumer.